includes/Modules/Analytics.php

```diff
@@ -626,6 +634,12 @@ protected function create_data_request( Data_Request $data ) {
 						}
 						return $option['internalWebPropertyID'];
 					};
+				case 'anonymize-ip-address':
+					return function() {
+						$default = true;
+						$option  = (array) $this->options->get( self::OPTION );
+						return isset( $option['anonymizeIPAddress'] ) ? (bool) $option['anonymizeIPAddress'] : $default;
```

Comment:
Super minor, but I don't think we need a `$default` variable here, we can just pass the `true`.

Reply:
I like the explicit nature of declaring it as `$default`; it makes this more readable so I'd rather leave it.

Reply:
In the future the default will be defined by the setting definition itself so any default handling here is temporary.